### PR TITLE
Updated the README

### PR DESCRIPTION
**Pull request by viccon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,25 +59,25 @@ let g:lightline.component_type = {
 
 " Add the components to the lightline:
 let g:lightline.active = {
-  \   left': [[ 'lsp_info', 'lsp_hints', 'lsp_errors', 'lsp_warnings', 'lsp_ok' ], [ 'lsp_status' ]]
+  \   'left': [[ 'lsp_info', 'lsp_hints', 'lsp_errors', 'lsp_warnings', 'lsp_ok' ], [ 'lsp_status' ]]
   \ }
 ```
```
Comment (josa42) [inline README.md:64]: 👍🏻 Thanks!

File: README.md [hunk 2]
```diff
@@ -59,25 +59,25 @@ let g:lightline.component_type = {
...
-- `g:lightline#lsp#indicator_info`
+- `let g:lightline#lsp#indicator_info`
   The indicator to use when there are information messages. Default is `~`.
 
-- `g:lightline#lsp#indicator_hints`
+- `let g:lightline#lsp#indicator_hints`
   The indicator to use when there are hints. Default is `>`.
 
-- `g:lightline#lsp#indicator_ok`
+- `let g:lightline#lsp#indicator_ok`
```
Comment (josa42) [inline README.md]: Could you revert these changes? I don't think should be in here.

**Review by josa42 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (viccon) [reply README.md]: Done

**Comment by josa42** [comment]
Thank you, @creativecreature!
